### PR TITLE
editing md to make danny happy

### PR DESCRIPTION
**Pull request by rajramasawmy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ The data.js file can be edited to personalise the page, messages and links.
 Add your own images (.jpg) to the backgroundPhotos folder, add the name of the images to the data.js file afterwards. 
 
 ## Tree
+<details><summary></summary>
```
Comment (dannyramasawmy) [inline README.md]: What does this summary tag do?

Comment (rajramasawmy) [reply README.md]: I dont know, it doesnt work without it I dont think.. got it from stackexchange probably..

**Review by dannyramasawmy — CHANGES_REQUESTED** [review]
Pretend to make an edit.

**Comment by rajramasawmy** [comment]
Tested md without summary tags, leaves a "details" text in the readme.

**Review by dannyramasawmy — APPROVED** [review]

**Pull request merged**